### PR TITLE
add `extraAttributes` in Navigation Group

### PR DESCRIPTION
**Pull request by atmonshi**
## Description

Adding support for `extraAttributes`, in NavigationGroup. I need to pass some CSS classes to specific navigation group 

example usage:
```php
return $panel
->navigationGroups([
        NavigationGroup::make()
            ->label('App')
            ->extraAttributes(['class'=>'highlighted']),
        ])
```

- [ ] Visual changes (if any) are shown using screenshots/recordings of before and after.

## Code style

- [x] `composer cs` command has been run.

## Testing

- [x] Changes have been tested.

## Documentation

- [x] Documentation is up-to-date.

### CODE REVIEW

**Comment by atmonshi** [comment]
running `composer cs` cases weird issues and some code deleted! so I revert it

File: packages/panels/resources/views/components/sidebar/index.blade.php
Comment (danharrin) [inline packages/panels/resources/views/components/sidebar/index.blade.php:1]: Please pass the attributes to topnav groups too

Comment (danharrin) [reply packages/panels/resources/views/components/sidebar/index.blade.php:1]: And anywhere else we use the group component, if we do

Comment (atmonshi) [reply packages/panels/resources/views/components/sidebar/index.blade.php:1]: the sidebar navigation (in cluster and sub navigation) dose use the `sidebar.group` but there is no way (I know of) to configure the group from the provider `navigationGroups`

Comment (danharrin) [reply packages/panels/resources/views/components/sidebar/index.blade.php:1]: Let's do it anyway, I think it is possible just requires a bit of hacking

**Comment by atmonshi** [comment]
all done 👍

File: packages/panels/resources/views/components/page/sub-navigation/sidebar.blade.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 >
     @foreach ($navigation as $navigationGroup)
         <x-filament-panels::sidebar.group
+            :attributes="\Filament\Support\prepare_inherited_attributes($navigationGroup->getExtraAttributeBag())"
```
Comment (zepfietje) [inline packages/panels/resources/views/components/page/sub-navigation/sidebar.blade.php]: Please move `:attributes` to be the last one in the list of attributes (like we do elsewhere).

**Review by zepfietje — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
I've changed my mind, the attributes that apply to a topnav dropdown would probably not be useful to apply to a sidebar group, which is used on mobile devices. Let's replace `extraAttributes()` with `extraSidebarAttributes()` and `extraTopbarAttributes()`?

**Comment by atmonshi** [comment]
hey @danharrin .

np, should I create a new `trait` for `extraSidebarAttributes` and `extraTopbarAttributes`?
or add the methods in the `NavigationGroup` class

**Comment by danharrin** [comment]
Please create new traits :)

**Comment by atmonshi** [comment]
done, hope this what you have in mind! let me know.
also if can you double check the docs `06-navigation.md` if needs improvement :)

**Comment by danharrin** [comment]
Thanks

**Pull request merged**